src/pyfaf/Makefile.am

```diff
@@ -23,3 +23,8 @@ pyfaf_PYTHON = \
     ureport_compat.py
 
 pyfafdir = $(pythondir)/pyfaf
+
+__init__.py: __init__.py.in
+	sed -e 's|@fafversion@|'$$(cat ../../faf-version)'|' $< > $@
```

Comment:
https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/autoconf.html#Preset-Output-Variables

You should probably use `$(top_builddir)` here. Though non-srcdir builds are likely already broken, this could break them further.